conf/webwork2.mojolicious.dist.yml

```
@@ -231,3 +231,90 @@ debug:
 hardcopy:
   # If 1, don't delete temporary files created when a hardcopy is generated.
   preserve_temp_files: 0
+
+# External Programs
+
+# These applications are often found in /bin, but sometimes in /usr/bin or even in /opt/local/bin.
+# You can use "which tar"  for example to find out where the "tar" program is located
+
+externalPrograms:
+  mv: /bin/mv
+  cp: /bin/cp
+  rm: /bin/rm
+  mkdir: /bin/mkdir
+  tar: /bin/tar
+  gzip: /bin/gzip
+  git: /bin/git
+  curl: /usr/bin/curl
+  mysql: /usr/bin/mysql
+  mysqldump: /usr/bin/mysqldump
+
+  # latex and related utilities
+  latex: /usr/bin/latex --no-shell-escape
+  pdflatex: /usr/bin/pdflatex --no-shell-escape
+
+  # Note that --no-shell-escape is important for security reasons.
+  # Consider using xelatex instead of pdflatex for multilingual use, and
+  # use polyglossia and fontspec packages (which require xelatex or lualatex).
+  # pdflatex: /usr/bin/xelatex --no-shell-escape
+
+  dvipng: /usr/bin/dvipng
+  convert: /usr/bin/convert
+  dvisvgm: /usr/bin/dvisvgm
+  pdf2svg: /usr/bin/pdf2svg
+
+  giftopnm: /usr/bin/giftopnm
+  ppmtopgm: /usr/bin/ppmtopgm
+  pnmtops: /usr/bin/pnmtops
+  pnmtopng: /usr/bin/pnmtopng
+  pnmtopnm: /usr/bin/pnmtopnm
+
+# Database options
+
+database:
+  # uncomment to allow database debug information
+  # debug: 1
+
+  # Select the desired DB driver: either 'mysql' or 'MariaDB'
+  # database_driver: mysql
+  driver: MariaDB
+
+  host: localhost
+  port: 3306
+
+  # name of the database
+  name: webwork
+
+  # Standard permissions command used to initialize the webwork database
+  # GRANT SELECT, INSERT, UPDATE, DELETE, CREATE, ALTER, DROP, INDEX, LOCK TABLES ON webwork.* TO webworkWrite@localhost IDENTIFIED BY 'passwordRW';
+  # where webworkWrite and passwordRW must match the corresponding variables in the next section.
+  username: ''
+  password: ''
```

Another thing here.  Change this to
```suggestion
  password: passwordRW
```
There are two reasons this should be done.  First, this is the current default in `site.conf.dist`.  Second, this shows that the quotes are not needed.

Although, documentation in the comments is needed here similar but different than what was in site.conf.dist.  In many cases quoting will not be needed here.  However, there are some cases where it will be needed.  The comments here should make this clear.  See https://symfony.com/doc/current/reference/formats/yaml.html for when quoting is needed in a YAML file.